alacritty_terminal/src/term/mod.rs

```diff
@@ -816,14 +816,17 @@ pub struct Term<T> {
     dynamic_title: bool,
 
     /// Clipboard access coupled to the active window
-    clipboard: Clipboard,
+    clipboard: Box<dyn Clipboard>,
```

Comment:
Do we have to use dynamic dispatch here? I'd rather stick with static dispatch whenever possible.

With the `Term<T, C = NopClipboard>` syntax it might be possible to prevent having to edit a bunch of unrelated code.

Reply:
I don't see any issue with dynamic dispatch for clipboard, but we can use static one here, sure.

Reply:
I've updated to use static dispatch, but I don't really like how it looks and how much it requires to change/carry around. Dynamic dispatch for clipboard made much more sense, since it's not something critical for performance and was integrated naturally.